guides/15_Background_Filter_Video_Processor.md

```diff
@@ -1,46 +1,46 @@
-# Integrating background blur into your Amazon Chime SDK for JavaScript application
+# Integrating background filters into your Amazon Chime SDK for JavaScript application
 
-## What is background blur?
+## What is a background filter?
 
-The background blur API allows builders to enable background blur on a video stream. To add a background blur to a video stream the builder needs to create a `VideoFrameProcessor` using `BackgroundBlurVideoFrameProcessor` and then insert that processor into a `VideoTransformDevice`. The video frame processor uses a TensorFlow Lite (TFLite) machine learning (ML) model along with JavaScript Web Workers and WebAssembly (WASM) to apply blur to the background of each frame in the video stream. These assets are downloaded at runtime when the video frame processor is created and not provided in the source directly.
+The background filter API allows builders to enable background filtering on a video stream. You can select from background blur filtering or replacement filter processors. To add a background filter to a video stream the builder needs to create a `VideoFrameProcessor` using `BackgroundBlurVideoFrameProcessor` or `BackgroundReplacementVideoFrameProcessor` and then insert that processor into a `VideoTransformDevice`. The video frame processor uses a TensorFlow Lite (TFLite) machine learning (ML) model along with JavaScript Web Workers and WebAssembly (WASM) to apply the filter to the background of each frame in the video stream. These assets are downloaded at runtime when the video frame processor is created and not provided in the source directly.
 
-Background blur is integrated into the [browser demo](https://github.com/aws/amazon-chime-sdk-js/tree/main/demos/browser) application. To try it out, launch the demo with `npm run start`, 
-join the meeting, click on the camera icon to enable video, then select the video filter drop down and select `Background Blur`.
+Background blur and replacement are integrated into the [browser demo](https://github.com/aws/amazon-chime-sdk-js/tree/master/demos/browser) application. To try it out, launch the demo with `npm run start`, join the meeting, click on the camera icon to enable video, then select the video filter drop down and select `Background Blur` or `Background Replacement`.
 
-## Background blur on the web
+## Background filtering on the web
 
-Background blur runs in the end user’s web browser, leveraging modern web platform features including WebAssembly and SIMD. See the section “[Can I use background blur in my application?](#can-i-use-amazon-background-blur-in-my-application)” for browser version requirements.
+Background filtering runs in the end user’s web browser, leveraging modern web platform features including WebAssembly and SIMD. See the section “[Can I use background filtering in my application?](#can-i-use-amazon-background-filtering-in-my-application)” for browser version requirements.
```

Comment:
`#can-i-use-amazon-background-filtering-in-my-application` contains `amazon`, but the actual section title (line #16) does have `amazon`. You can check it on a preview mode.

<img src="https://user-images.githubusercontent.com/36240708/148454465-92c24fc7-7fbb-4a75-90a1-e707dc175d3e.png" width="200">

Reply:
fixed